iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/tests/iothub/methods/DeviceMethodTests.java

```diff
@@ -43,7 +44,7 @@ public void cleanToStart() throws Exception
     {
         super.cleanToStart();
     }
-
+/*
     @Test
```

Comment:
Why is this commented?

Reply:
Whoops, my mistake. I can uncomment these now